### PR TITLE
fix: delete conversation option is displayed for non team conversations

### PR DESCRIPTION
**Pull request by MohamadJaara**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
    - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
    - [ ] contains a reference JIRA issue number like `SQPIT-764`
    - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
    - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Issues

delete conversation option is displayed for non team conversations

### Solutions

display the delete conversation option if and only if the conversation have the following criteria:
- [x] is team conversation.
- [x] self is the creator.
- [x] self is still a member.
- [x] conversation is a group conversation.
- [ ] self is Admin(this case is still not covered to avoid over complicating the PR since it require changes in the conversation view).

Needs releases with:

- [ ] GitHub link to other pull request

#### Test Coverage (Optional)

- [ ] I have added automated test to this contribution

#### How to Test

_Briefly describe how this change was tested and if applicable the exact steps taken to verify that it works as expected._

### Notes (Optional)

_Specify here any other facts that you think are important for this issue._

### Attachments (Optional)

_Attachments like images, videos, etc. (drag and drop in the text box)_ 

----
#### PR Post Submission Checklist for internal contributors (Optional)

- [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

- [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-1206/1/) **failed**.

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-1206/2/) **failed**.

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-1206/3/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(5110579)(5110579)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(5110579)(5110579)-dev-release.apk)

File: app/src/main/kotlin/com/wire/android/navigation/HomeNavigation.kt [hunk 1]
```diff
@@ -65,7 +65,7 @@ enum class HomeNavigationItem(
                     onSnackBarStateChanged = homeState::setSnackBarState,
                     searchBarState = homeState.searchBarState,
                     isBottomSheetVisible = homeState::isBottomSheetVisible
-                )
```
Comment (tmspzz) [inline app/src/main/kotlin/com/wire/android/navigation/HomeNavigation.kt]: ?

**Comment by tmspzz** [comment]
@MohamadJaara submodule has changed. Is there a corresponding PR?

File: app/src/main/kotlin/com/wire/android/navigation/HomeNavigation.kt (hunk 1 above)
Comment (MohamadJaara) [reply app/src/main/kotlin/com/wire/android/navigation/HomeNavigation.kt]: Styling issue, I fixed it

**Comment by AndroidBob** [comment]
Build [7](https://10.10.124.134/job/wire-android-reloaded/job/PR-1206/7/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(5126844)(5126844)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(5126844)(5126844)-dev-release.apk)

**Pull request merged**